server/services/goal-service.js

```diff
@@ -16,11 +15,31 @@ function get(req, res) {
     })
 }
 
+// function getOnMYid (req, res) {
+//   console.log(req);
+//   const {userId, mmyyId} = req.params;
```

Comment:
Maybe let's look into deleting this if applicable?

Reply:
Done!